src/main/java/core/basesyntax/strategy/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.strategy;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private final String operation;
+
+    Operation(String operation) {
+        this.operation = operation;
+    }
+
+    public String getOperation() {
+        return operation;
+    }
+
+    public static Operation getByCode(String character) {
```

Comment:
```suggestion
    public static Operation getByCode(String code) {
```
don't name variables as other data types, may be confusing